docs/changelog.md

```diff
@@ -10,7 +10,7 @@ title: Changelog
 * [FEATURE] `DataContextVariables` CRUD for `concurrency` (#5273)
 * [FEATURE] `DataContextVariables` CRUD for `progress_bars` (#5274)
 * [FEATURE] Integrate `DatasourceStore` with `DataContext` (#5292)
-* [FEATURE] Support both UserConfigurableProfiler and OnboardingDataAssistant in "CLI SUITE NEW --PROFILE <name>" command (#5306)
+* [FEATURE] Support both UserConfigurableProfiler and OnboardingDataAssistant in "CLI SUITE NEW --PROFILE" command (#5306)
```

Comment:
@Shinnnyshinshin What do we do regarding being able to provide the actual name (name is not command -- command is not needed) -- and it should all read in lower case.  Thanks!